ArduCopter/GCS_Mavlink.cpp

```diff
@@ -329,13 +329,6 @@ bool GCS_MAVLINK_Copter::try_send_message(enum ap_message id)
 #endif // MOUNT == ENABLED
         break;
 
-    case MSG_OPTICAL_FLOW:
-#if OPTFLOW == ENABLED
-        CHECK_PAYLOAD_SIZE(OPTICAL_FLOW);
-        send_opticalflow(copter.optflow);
-#endif
-        break;
-
```

Comment:
sigh about convention of splitting per-dir changes :( - "move", but where's the rest?